content/en/release-notes/_index.md

```diff
@@ -12,6 +12,53 @@ version: 'releases'
 title: Releases
 ---
 
+# Apache Log4j announcement
+
+Following news of [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
```

Comment:
In response to the [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), R3 is releasing patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM.